landing-page.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>This is simple landing page</h1>
```

Comment:
Can you please make this beautiful
```suggestion
    <h1>This is simple landing page</h1>
```